dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHV1IT.java

```diff
@@ -227,6 +230,21 @@ void testToolsIdVersionsVersionIdTypeDockerfile() {
         assertTrue(!responseObject.getDockerfile().isEmpty() && !responseObject.getUrl().isEmpty());
     }
 
+    /*
+    Checks the GA4GHv1 endpoints to make sure the CORS header is present
+     */
+
+    @Test
+    void testV1CorsHeader() {
+        final String origin = "http://mysite.org";
+        String nginxRewrittenPath = TestUtility.mimicNginxRewrite(baseURL + "tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/dockerfile", basePath);
+        MultivaluedMap<String, Object> headers = new MultivaluedHashMap<>();
```

Comment:
Think you basically have copy-paste code in two forms for these five lines, one for present one for not-present. 
Could make into methods

Reply:
Thanks yeah I was curious what people thought of that. If we end up having to alter these tests or increase the coverage I'll go back and do this.